ebuild-writing/common-mistakes/text.xml

```diff
@@ -380,6 +380,19 @@ Make sure when you bump a version, the stable KEYWORDS are all marked as
 
 </body>
 </subsection>
+<subsection>
+<title>Unused flags in IUSE</title>
+<body>
+
+<p>
+Sometimes vestigal USE flags remain in IUSE despite having no function. Remember
+to prune these.
```

Comment:
Maybe mention CI tools that would enable something like this to be caught before committing?

Reply:
No tools cover this as far as I know. They cover leftover flags in metadata.xml but not in the ebuild itself which are noops (it'd be tricky to analyse this right now given it'd involve checking eclasses too).